prebid/managed.md

```diff
@@ -12,7 +12,7 @@ Several Prebid.org members will configure and host your Prebid implementations,
 
 <table class="cellpadding">
 <tr><td><a href="https://www.appnexus.com/en/publishers/header-bidding"><img src="/assets/images/partners/founders/appnexus.png" width="100"></a></td>
-<td>Offers a full set of managed solutions. Get started at <a href="https://www.appnexus.com/en/publishers/header-bidding">https://www.appnexus.com/en/publishers/header-bidding</a></td>
+<td>AppNexus, a Xandr company, offers a full managed Prebid solution including implementation and maintenance, as well as consulting support for publishers with their own tech resources. We work with you to enable client and/or server-side header bidding across web, AMP, and mobile app channels, and support display, video, and native ad formats. <a href="https://www.appnexus.com/contact">Contact us</a>for more information, and to get a quote! </td>
```

Comment:
This isn't an AppNexus web site, so I'd remove the "we" and "us" terminology. "AppNexus works with you..." and "Contact your AppNexus representative..."